js/main.js

```diff
@@ -28,9 +32,53 @@ $entryForm.addEventListener('submit', function (event) {
     entryNotes: $entryNotes.value,
     entryId: data.nextEntryId++
   };
-  data.entries.unshift(entryFormObject);
+  data.entries.push(entryFormObject);
```

Comment:
You should be adding to the beginning of the array, not the end, so that your data is sorted from newest to oldest.

Reply:
Oh, so I got it right the first time around, xD